src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,52 @@
 package core.basesyntax;
 
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        try {
+            List<String> linesFromFile = Files.readAllLines(Path.of(fromFileName));
+            DataRecord dataRecord = new DataRecord();
+            for (String line : linesFromFile) {
+                String[] splitStrings = line.split(",");
+                if (splitStrings[0].equals("supply")) {
+                    dataRecord.addSupply(Integer.parseInt(splitStrings[1]));
+                } else if (splitStrings[0].equals("buy")) {
+                    dataRecord.addBuy(Integer.parseInt(splitStrings[1]));
+                }
+            }
+            String supplyText = "supply"
```

Comment:
Use a `StringBuilder` instead of `String` concatenation in loops. It's more memory-efficient especially when concatenating strings in a loop.